src/pages/briefs/index.tsx

```diff
@@ -449,6 +478,40 @@ const Briefs = (): JSX.Element => {
             </div>
           ))}
         </div>
+        <Pagination
+          pageSize={3}
```

Comment:
looks too small
should be able to select number of items to show on a page.

Reply:
the items are large and 3 is more than enough for a single page scroll

Reply:
sure